sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -455,6 +461,9 @@ annotated_ptr& operator++() noexcept;
 |
 Prefix increment operator.
 
+If property list contains properties that are not sustainable through pointer
+arithemic, this operator errors out.
```

Comment:
Maybe it's just my bias, but when I read "this operator errors out" it sounds like a run-time exception.  I believe you intend for this to be a compile-time error.  Can you make that more clear?  For example:

```suggestion
If property list contains properties that are not sustainable through pointer
arithmetic, this operator is not valid and if it is used a diagnostic is required.  
```